src/ray/gcs/gcs_server/gcs_autoscaler_state_manager.cc

```diff
@@ -131,7 +128,8 @@ void GcsAutoscalerStateManager::GetPendingGangResourceRequests(
   }
 
   // Iterate through each placement group load.
-  for (const auto &pg_data : placement_group_load->placement_group_data()) {
+  for (auto &&pg_data :
+       std::move(*placement_group_load->mutable_placement_group_data())) {
     auto gang_resource_req = state->add_pending_gang_resource_requests();
     auto pg_state = pg_data.state();
```

Comment:
if you want, you could move or const ref

Reply:
it's an enum

Reply:
Ah got it.. I usually just ref in 99% of the cases, don't want to think twice on pod vs non-pod :sweat_smile: